src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/SelectAttributeType.php

```diff
@@ -26,6 +26,19 @@
 final class SelectAttributeType extends AbstractType
 {
     /**
+     * @var string $defaultLocale
+     */
+    private $defaultLocale;
+
+    /**
+     * @param string $defaultLocale
+     */
+    public function __construct(string $defaultLocale)
+    {
+        $this->defaultLocale = $defaultLocale;
```

Comment:
`SelectAttributeChoicesCollectionType` uses `LocaleProviderInterface::getDefaultLocaleCode()` to get its default locale, this one has base locale injected as `$defaultLocale`, is that intentional?

Reply:
Anyway, if it's intentional, I guess `$baseLocale` would be more appropriate name then.